### PR TITLE
Switch functionality for as3 and cccl using agent param

### PR DESCRIPTION
**Pull request by srivastavnitin24**

### CODE REVIEW

File: cmd/k8s-bigip-ctlr/main.go [hunk 1]
```diff
@@ -181,6 +182,9 @@ func _init() {
 		"Optional, when set to true, enable insecure SSL communication to BIGIP.")
 	trustedCertsCfgmap = bigIPFlags.String("trusted-certs-cfgmap", "",
 		"Optional, when certificates are provided, adds them to controller’s trusted certificate store.")
+	//TODO:
```
Comment (ssurenr) [inline cmd/k8s-bigip-ctlr/main.go]:
```suggestion
	// TODO: Rephrase agent functionality
```

File: pkg/appmanager/as3Manager_test.go [hunk 2]
```diff
@@ -277,5 +277,19 @@ var _ = Describe("As3Manager Tests", func() {
 			Expect(reflect.DeepEqual(origCfg, generatedCfg)).To(BeTrue(), "Failed to Create JSON with correct configuration")
 		})
 	})
+	Describe("postRouteDeclarationHost", func() {
```
Comment (ssurenr) [inline pkg/appmanager/as3Manager_test.go]: Let's keep this change in the appropriate PR

File: cmd/k8s-bigip-ctlr/main.go (hunk 1 above)
Comment (srivastavnitin24) [reply cmd/k8s-bigip-ctlr/main.go]: Changes done

File: pkg/appmanager/as3Manager_test.go (hunk 2 above)
Comment (srivastavnitin24) [reply pkg/appmanager/as3Manager_test.go]: Sure @ssurenr will take the test case in an appropriate PR.

**Pull request merged**